### ISSUE TITLE
Run the only ROM inside a folder without entering it 

### ISSUE THREAD

**Issue by joeelliott** [Aloshi/EmulationStation#227]
could you include folders as being considered like a filename

example:

/usr/games/nintendo/Top-Gun-US.nes/Top-Gun-US.nes

the first Top-Gun-US.nes be listed as the game, or even the second, doesnt matter.  

This paves the way for a cleaner system structure for roms (a la Higan) like
/usr/games/Top-Gun.nes/program.rom
/usr/games/Top-Gun.nes/cheats.bml
/usr/games/Top-Gun.nes/screenshot-0001.png
/usr/games/Top-Gun.nes/state-1.bst
/usr/games/Top-Gun.nes/save.ram

If we begin supporting the structure then it makes for a much better project that is becoming its own distribution entirely.

**Comment by ThomasHineXYZ** [comment]
Agreed. This would be nice to have. Especially for Wii and Gamecube game rips.
